stackslib/src/net/api/gettransaction.rs

```diff
@@ -0,0 +1,197 @@
+// Copyright (C) 2013-2020 Blockstack PBC, a public benefit corporation
+// Copyright (C) 2020-2023 Stacks Open Internet Foundation
```

Comment:
It's 2024.

Reply:
please ignore this file, is just a copy/paste for the draft, i am going to push a cleaned version as soon as i move it out from draft status

Reply:
same for the following comments, i am leaving them opened just to avoid missing something before the "undrafting"